### PR TITLE
Add a license of cubism, cc #9248

### PR DESCRIPTION
**Pull request by clairetsai818**
https://github.com/square/cubism/blob/master/LICENSE

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@clairetsai818 You added license at wrong place. That place is still in `npmFileMap` field, but you need to add license as new field.

**Comment by clairetsai818** [comment]
@pvnr0082t okay, thanks.

File: ajax/libs/cubism/package.json [hunk 1]
```diff
@@ -29,5 +29,6 @@
         "cubism.v1.min.js"
       ]
     }
-  ]
+  ],
+  "license": "Square"
```
Comment (PeterDaveHello) [inline ajax/libs/cubism/package.json]: Square is the company name, not the license.

**Comment by PeterDaveHello** [comment]
Better now but still wrong, @clairetsai818 take a look at the SPDX license list for the valid license identifier:
https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json

**Pull request merged**